# name: catch ( ... ) { ... }
# key: catch
# --
catch (${exception}) {
    $0
}